src/index.ts

```diff
@@ -37,11 +37,6 @@ export const configureMonacoTailwindcss: typeof import('monaco-tailwindcss').con
         languageSelector,
         createHoverProvider(workerManager.getWorker),
       ),
-      registerMarkerDataProvider(
-        { editor },
```

Comment:
If I see this correct - this was a duplicate?

Reply:
Yep. I believe I added it here initially for local testing, but then added the loop below later to support other languages.